tests/dummy/app/templates/docs/rules.md

```diff
@@ -0,0 +1,16 @@
+# Rules for Data-Dependent Animations
+
+`rules` choose which `transition` to run when data changes in a list that is being animated. To do this, `rules` compares the new data value to the old data value (`oldList` and `newList`). If a component has `rules` and a `transition`, the `rules` will take precedence over the `transition`. 
```

Comment:
The first sentence could maybe be stated more simply: "To use different animated transitions under different conditions, a `rule` can be defined. For example, (describe use case)".